easycord/plugins/polls.py

```
@@ -39,15 +53,19 @@ def _tally(self) -> list[int]:
     def build_embed(self, *, closed: bool = False) -> discord.Embed:
         counts = self._tally()
         total = sum(counts) or 1
-        lines = []
-        for opt, count in zip(self.options, counts):
-            pct = count / total
-            bar = self._bar(round(pct * 10))
-            votes_str = f"{count} vote{'s' if count != 1 else ''} ({pct:.0%})"
-            lines.append(f"**{opt}**\n`{bar}` {votes_str}")
+        lines = [
+            self._format_option_line(option, count, total)
+            for option, count in zip(self.options, counts)
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)